index.html

```diff
@@ -25,6 +25,12 @@
                 <div class="rectangle"></div>
             </div>
```

Comment:
- [ ] Please try to stick Figma design and update the font size of the heading (Felix Odete)

Your design  | Figma design
------------- | -------------
<img width="703" alt="Screenshot 2022-03-31 at 16 29 30" src="https://user-images.githubusercontent.com/77038610/161054953-adeef07d-364f-4a5c-8b56-31c6fe7ab2b4.png"> | <img width="438" alt="Screenshot 2022-03-31 at 16 29 45" src="https://user-images.githubusercontent.com/77038610/161054997-039f3a9f-dd50-4fbe-b9c0-843d63b5ac22.png">
- Please decrease the font size to 110px as it is provided in Figma. and make sure that the line height is the same as in Figma

- [ ] Please change the color of menu items to black as provided in Figma:

Your design  | Figma design
------------- | -------------
<img width="430" alt="Screenshot 2022-03-31 at 16 32 20" src="https://user-images.githubusercontent.com/77038610/161055386-e4e0eec7-60ec-4a93-aef3-3005fcc64abc.png"> | <img width="363" alt="Screenshot 2022-03-31 at 16 32 37" src="https://user-images.githubusercontent.com/77038610/161055446-a1a253b9-edc2-4905-88b2-b7174801cf9f.png">

- [ ] After you complete doing changes, please make sure that you have the same margins, paddings and colors as in original design